avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1097,6 +1099,43 @@ def removePart(particles, mask, nRemove):
     return particles
 
 
+def savePartToPickle(dictList, outDir):
+    """ Save each dictionary from a list to a pickle in outDir; works also for one dictionary instead of list
+
+        Parameters
+        ---------
+        dictList: list or dict
+            list of dictionaries or single dictionary
+        outDir: str
+            path to output directory
+    """
+    if len(dictList) > 1:
+        for dict in dictList:
+            pickle.dump(dict, open(os.path.join(outDir, "particles%f.p" % dict['t']), "wb"))
+    else:
+        pickle.dump(dictList, open(os.path.join(outDir, "particles.p"), "wb"))
+
+
+def readPartFromPickle(inDir, flagAvaDir=False):
+    """ Read pickles within a directory and return List of dicionaries read from pickle """
```

Comment:
Please document the flagAvaDir override (explain what it does..)

Reply:
done